src/core/StelCore.hpp

```diff
@@ -74,7 +74,7 @@ class StelCore : public QObject
 	enum FrameType
 	{
 		FrameUninitialized,			//!< Reference frame is not set (FMajerech: Added to avoid condition on uninitialized value in StelSkyLayerMgr::draw())
-		FrameAltAz,				//!< Altazimuthal reference frame centered on observer.
+		FrameAltAz,				//!< Altazimuthal reference frame centered on observer: +x=south, +y=east, +z=zenith.
```

Comment:
Correct. See also external use in http://stellarium.org/doc/head/remoteControlApi.html#rcMainServiceView